src/anubhav/BankDebitCredit.java

```diff
@@ -0,0 +1,100 @@
+package anubhav;
+
+import java.security.PublicKey;
+
+/*Create a Banking System which has the following functionality.
+
+Create two object to perform below scenario.
+1) Method to debit amount
+2) Method to credit amount
+3) Method to printBalance [it will print current balance]
+4) individualTransactionSummary() method should show how many times individually debit, credit & printBalance method called .
+Hint : Create 3 non static instance variable.
+Output :
+user1 transaction summary : Credit - 2 times, Debit - 1 times, printBalance - 1 time
+user2 transaction summary : Credit - 5 times, Debit - 2 times, printBalance - 0 time
+
+5) allTransactionSummary() method should show by both users total how many times debit, credit & printBalance method get called.
+Hint : Create 3 static variables will be required.
+Output :
+All transaction summary : Credit - 7 times, Debit - 3 times, printBalance - 1 time*/
+
+public class BankDebitCredit {
+	
+	String username;
+	int totalBalance;
+	int debitCount;
+	int creditCount;
+	int printBalance;
+	static int totalNoOfCredits;
+	static int totalNoOfDebits;
+	static int totalNoOfTimesPrintBalance;
+	
+	void customerInfo (String uname, int currentBalance){
+		totalBalance= currentBalance;
+		username= uname;
+	}
+	
+	void debit (int dbAmount){
+		while (dbAmount<=totalBalance)
```

Comment:
why you are using this while loop here? 
you should use if condition.

Please change,